python/py-roman/Portfile

```diff
@@ -1,41 +1,37 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
-PortGroup python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
 
-set realname        roman
-name                py-${realname}
-version             2.0.0
-python.versions     26 27 33 34 35 36
+name                py-roman
+version             3.0
 categories-append   textproc
 license             PSF-2.1.1
 platforms           darwin
 supported_archs     noarch
 maintainers         {aronnax @lpsinger} openmaintainer
+
 description         Integer to Roman numerals converter
 long_description    ${description}
-homepage            https://pypi.python.org/pypi/${realname}/
-master_sites        pypi:[string index ${realname} 0]/${realname}/
-distname            ${realname}-${version}
-use_zip             yes
-
-checksums           md5     aa71d131eec16d45c030fd06a27c9d17 \
-                    sha1    7fd136f80cf780bbf802ffe1c4989a64ff469c75 \
-                    rmd160  a8ca66d5fafd340986a45332d861d6bae259fe1e
-
-if {${name} eq ${subport}} {
-    livecheck.type  regex
-    livecheck.regex [format "%s-%s" ${realname} {(\d+(?:\.\d+)*)}]
-} else {
-    depends_build   port:py${python.version}-setuptools
 
-    pre-activate {
-        if {[file exists ${python.pkgd}/roman.py]
-            && ![catch {set vers [lindex [registry_active py${python.version}-docutils] 0]}]
-            && [vercmp [lindex $vers 1] 0.6] < 0} {
-            registry_deactivate_composite py${python.version}-docutils "" [list ports_nodepcheck 1]
-        }
-    }
+homepage            https://pypi.python.org/pypi/${python.rootname}/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  39d72b580e07e5c49bdf3a7db9efe987f3ff8d60 \
+                    sha256  1cb1cfc9386ec39bd5dd3190c2c89c1b253140bccda16807aab3627cbb3f275d \
+                    size    5184
+
+python.versions     26 27 33 34 35 36 37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    test.run        yes
+    test.cmd        ${python.bin} setup.py
 
     livecheck.type  none
+} else {
+    livecheck.type  pypi
```

Comment:
AFAIK there is no need to explicitly set `livecheck.type pypi` since it automatically inferred from `master_sites pypy:...`.

Reply:
true. That patterns is present though in many Portfiles, but I agree that is simplifies thing not to have extra, unnecessary  'else' statements and have removed it.